TheJumpCloud/jcapi-powershell
# Update SDK V2 to address read only o365 property

## Issues
* [SA-3001](https://jumpcloud.atlassian.net/browse/SA-3001) - Remove Read Only property for Office 365 integrations

## What does this solve?

Addressed an issue in `Update-JcSdkOffice365` where the `UserLockoutAction` and `User PasswordExpirationAction` values could not be set

## Is there anything particularly tricky?

## How should this be tested?

`Update-JcSdkOffice365` will now let you update the `UserLockoutAction` and `User PasswordExpirationAction` values. Build the V2 module on your own and you should be able to update the fields with a request like the following:
![Screenshot 2023-11-29 at 12 04 27 PM](https://github.com/TheJumpCloud/jcapi-powershell/assets/54448601/8db98276-95ce-4a62-9fe4-d678e03c65ab)


## Screenshots

[SA-3001]: https://jumpcloud.atlassian.net/browse/SA-3001?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
